index.html

```diff
@@ -0,0 +1,43 @@
+<!DKunkel pr 1 -On Paper>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css"
+  </head>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Courgette&family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&display=swap"
+    rel="stylesheet"
+  />
```

Comment:
This `link` element belongs inside your `head`, like so:
```suggestion
  <link
    href="https://fonts.googleapis.com/css2?family=Courgette&family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&display=swap"
    rel="stylesheet"
  />
  </head>
```

Reply:
That's what I thought, but for some reason it wouldn't link to the CSS and apply the fonts until I tried placing it outside the head ?

Reply:
I spotted it! So, on line 8 you have a bit of incomplete HTML:
`<link rel="stylesheet" href="style.css"`

If you close out that `link` element:
`<link rel="stylesheet" href="style.css">`
(see the missing `>` bracket in the first example?)
and _then_ move your additional `link` element into the `head`, everything should work as expected 👍🏻